packages/blockchain-wallet-v4/src/walletCrypto/index.spec.js

```diff
@@ -148,7 +151,9 @@ describe('WalletCrypto', () => {
     })
     it('should fail because of wrong password', done => {
       wCrypto.decryptWallet('wrong password', data.v3).fork(failure => {
-        expect(failure).toEqual('v2v3: wrong_wallet_password')
+        expect(failure.message).toEqual(
+          'Unsupported state or unable to authenticate data'
+        )
         done()
       }, done)
     })
```

Comment:
Looks like these are decryption unit tests? Can I suggest two more test cases:

1. Corrupted ciphertext - take a correctly encrypted payload and tag, flip a single bit in the payload
2. Corrupted GCM tag - take a correctly encrypted payload and tag, flip a single bit in the tag

Expected result - should return exactly the same error as incorrect password